### PR TITLE
369 render diagrams

### LINKED ISSUE #369: Render diagrams
Use Github action to automatically render drawio diagrams to png files for embedding within docs as code. 

Plan is to use this well maintained action.

### PR DESCRIPTION
**Pull request by michaelpearsonHO**
resolves #369.

Tests performed on personal repository here https://github.com/michaelpearsonHO/sre-monitoring-as-code/actions/runs/3394158723

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -97,6 +97,18 @@ Run `bundle update` to make sure you're using the most recent Ruby gem versions.
 
 Run `bundle exec middleman build --verbose` to get detailed error messages to help with finding the problem.
 
+## Diagrams
+
+Diagrams are maintained in [DrawIO](https://app.diagrams.net) using the file [monitoring_as_code.drawio](https://github.com/HO-CTO/sre-monitoring-as-code/blob/369-render-diagrams/docs/monitoring_as_code.drawio). 
+Once landing in DrawIO web interface select `Save diagrams to: Device > Open Existing Diagram` and select [monitoring_as_code.drawio](https://github.com/HO-CTO/sre-monitoring-as-code/blob/369-render-diagrams/docs/monitoring_as_code.drawio) from you're local repository.
```
Comment (samiwelthomasHO) [inline docs/README.md:103]: Just a small nit pick, I think these links will break as soon as the branch is deleted.

**Review by samiwelthomasHO — APPROVED** [review]
Looks good to me, one small comment.

File: docs/README.md (hunk 1 above)
Comment (michaelpearsonHO) [reply docs/README.md:103]: My mistake, good spot. Thanks Sam

**Review by arifulhaqueHO — APPROVED** [review]
Looks good to me

**Pull request merged**